instagram/src/Components/PostContainer/Post.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from "react";
+
+import "./Post.css";
+
+import comment from "./comment.png";
+import heart from "./heart.png";
+import pinkheart from "./pinkheart.png";
+
+// import PostContainer from "./PostContainer";
+import CommentSectionConatiner from "../CommentSection/CommentSectionContainer";
+import PropTypes from "prop-types";
+
+// See if length of search str is greater than 0
+// If so, then filter the dummyData & check the search criteria for matches with same characters in username
+// if the resulting  array ahs a length greater than 0, some results were found so display those
+// if the resulting filtered\ array has a length of 0, no results were found so display that
+// if the search criteria's length is 0, we arent searching for anything so just display all posts
+
+// still need to fix error on the below code
+
+export default class Post extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      likes: this.props.post.likes,
+      liked: false
+    };
+  }
+
+  likePost = () => {
+    this.setState(prevState => ({
+      liked: !prevState.liked,
+      likes: !prevState.liked ? prevState.likes + 1 : prevState.likes - 1
+    }));
```

Comment:
this doesn't seem to like me at the moment... how's it running for you?

Reply:
its not working for me currently either :(